restli-tools/src/main/java/com/linkedin/restli/tools/data/ExtensionSchemaValidationCmdLineApp.java

```diff
@@ -112,44 +124,58 @@ public static void main(String[] args) throws Exception
       _logger.error("Invalid arguments: " + e.getMessage());
       System.exit(1);
     }
+    catch (InvalidExtensionSchemaException e)
+    {
+      _logger.error(e.getMessage());
```

Comment:
Maybe give some hint in the beginning, just like the previous parseexception, so when see the log, we know clearly this is from extension schema validator

Reply:
Fixed